packages/portal-proto/src/features/files/AssociatedCB.tsx

```diff
@@ -74,10 +74,13 @@ const AssociatedCB = ({
           : undefined;
 
       const annotationsLink = url ? (
-        <Link href={url} passHref>
-          <a className="text-utility-link underline" target="_blank">
-            {caseData.annotations.length}
-          </a>
+        <Link
+          href={url}
+          passHref
```

Comment:
do we still need `passHref` here?

Note: This is popping up in multiple places.  I'm going to stop commenting on each one.  Instead, there's two things to consider
1. Do we still need `passHref` in any place where we're removing an `<a>` tag?
2. Do we need to keep or add `passHref` in any place where we add `legacyBehavior`?

Reply:
This is a good point which I didn't think of. I will check this for all the links which were automatically changed using the [codemod](https://nextjs.org/docs/pages/building-your-application/upgrading/codemods#new-link) they provided.

Reply:
This has been fixed.